perf-tests/clusterloader2/pkg/util/multi-tenancy.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"os"
+	"strings"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog"
+	apimachineryvalidation "k8s.io/apimachinery/pkg/api/validation"
+)
+
+func GetTenant() string {
+	tenantName := os.Getenv("SCALEOUT_TEST_TENANT")
+    if len(tenantName) == 0 {
```

Comment:
Maybe parse the content of this environment variable to return []tenants? So we can set "EXPORT SCALEOUT_TEST_TENANTS=tenant1;tenant2;tenant3..."